src/main/java/jp/kusumotolab/kgenprog/fl/SpectrumBasedFaultLocalization.java

```diff
@@ -76,11 +80,16 @@ private boolean isSkippableFormula() {
           continue;
         }
 
-        final ASTLocation loc = locations.get(locations.size() - 1);
-        final long ef = testResults.getNumberOfFailedTestsExecutingTheStatement(path, loc);
-        final long nf = testResults.getNumberOfFailedTestsNotExecutingTheStatement(path, loc);
-        final long ep = testResults.getNumberOfPassedTestsExecutingTheStatement(path, loc);
-        final long np = testResults.getNumberOfPassedTestsNotExecutingTheStatement(path, loc);
+        final ASTLocation location = locations.get(locations.size() - 1);
+        if(registeredLocations.contains(location)){
+          continue;
+        }else{
+          registeredLocations.add(location);
+        }
```

Comment:
不要なelseですね。
あと、フォーマット(空白)されていないので、フォーマットお願いします。
```suggestion
        if(registeredLocations.contains(location)) {
          continue;
        }
        registeredLocations.add(location);
```